src/Provisioning.ts

```diff
@@ -31,11 +31,11 @@ type Param = string | { param: string, required: boolean};
 type Verbs = "getbotid"|"authurl"|"channels"|"getlink"|"link"|"logout"|"removeaccount"|"teams"|"accounts"|"unlink";
 
 // Decorator
-function command(...params: Param[]) {
```

Comment:
feels a big ugh here to have an arrow function for something that doesn't need access to the outside scope.